src/commands/addons/admin/manifest/diff.ts

```diff
@@ -49,11 +49,10 @@ export default class Diff extends CommandExtension {
     .catch((err: any) => {
       if (err){
         if (slug) {
-          this.error(`Unable to fetch manifest of a slug named ${color.blue(slug)}`)
+          this.error(`Unable to fetch manifest for add-on ${color.blue(slug)}`)
         } else {
-          this.error('Please make sure you have a slug.')
+          this.error('Please make sure you have a manifest in the current directory. New manifests have no history to diff.')
```

Comment:
@mathias can we make this error out earlier, if the slug is not defined we should not try to download

Reply:
Yeah. I'd have to look at it again, but I agree that it shouldn't try to download if it can't find the file locally or the file is missing an `id` field that we need to use.

I think I went down the rabbithole on this earlier and was confused about where the JSON parsing was happening on the Manifest file.